dockernet/config.sh

```diff
@@ -57,6 +57,13 @@ STWALK_DENOM="stuwalk"
 STEVMOS_DENOM="staevmos"
 STDYDX_DENOM="studydx"
 
+IBC_GAIA_CHANNEL_0_STATOM_DENOM='ibc/054A44EC8D9B68B9A6F0D5708375E00A5569A28F21E0064FF12CADC3FEF1D04F'
+IBC_GAIA_CHANNEL_1_STATOM_DENOM='ibc/8B21DA0E34A49AE151FEEBCCF3AFE1188E24BA8E19439FB93434DF6008E7E228'
+IBC_GAIA_CHANNEL_2_STATOM_DENOM='ibc/60CB7A5465C318C8F68F603D78721A2ECC1DA2D0E905C6AD9ACD1CAC3F0DB22D'
+IBC_GAIA_CHANNEL_3_STATOM_DENOM='ibc/0C0FD07C29EB075C18EA77B73CF9FCE68A268E0738C9F5B11D13E418AD889437'
```

Comment:
I think you can remove all besides channel 0. In dockernet, each host chain → stride should be channel 0

Reply:
handled in the redeem stake PR